HARK/ConsumptionSaving/ConsAggShockModel.py

```diff
@@ -1228,9 +1229,11 @@ def calcAFunc(self, MaggNow, AaggNow):
         # Plot aggregate resources vs aggregate savings for this run and print the new parameters
         if verbose:
             print('intercept=' + str(intercept) + ', slope=' + str(slope) + ', r-sq=' + str(r_value**2))
-            # plot_start = discard_periods
-            # plt.plot(logMagg[plot_start:],logAagg[plot_start:],'.k')
-            # plt.show()
+            #plot_start = discard_periods
+            #plt.xlabel('log(MaggNow)')
+            #plt.ylabel('log(AaggNow)')
+            #plt.plot(logMagg[plot_start:],logAagg[plot_start:],'.k')
+            #plt.show()
```

Comment:
If your goal is to 'clean up" this code, I'd suggest that removing commented code blocks would make the code 'cleaner'.